fuzzywuzzy/fuzz.py

```diff
@@ -202,9 +193,7 @@ def QRatio(s1, s2, force_ascii=True):
     p1 = utils.full_process(s1, force_ascii=force_ascii)
     p2 = utils.full_process(s2, force_ascii=force_ascii)
 
-    if not utils.validate_string(p1):
-        return 0
-    if not utils.validate_string(p2):
+    if not (utils.validate_string(p1) and utils.validate_string(p2)):
```

Comment:
Again no performance implication. Simply a preference, whether these two if statements should be joined